stl/inc/xhash

```diff
@@ -1915,6 +1915,116 @@ protected:
         return _List._Getal();
     }
 
+    struct _Multi_equal_check_result {
+        bool _Equal_possible;

```

Comment:
I am mildly uncomfortable with the lack of DMIs here; `return {}` initializes this `bool` to `false` but the risk of garbage-init is higher than I would like. Would it be better to add DMIs for `{false}` and `{}` for the iterator?

Reply:
In my tests all the functions returning this type were inlined so I have no objection to DMIs.